MdePkg/Include/Library/PerformanceLib.h

```diff
@@ -40,7 +40,9 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #define PERF_INMODULE_END_ID       0x41
 #define PERF_CROSSMODULE_START_ID  0x50
 #define PERF_CROSSMODULE_END_ID    0x51
-
+// MU_CHANGE

```

Comment:
For the UEFI variable changes, I suggest two commits in one PR.

- Commit 1: Changes to PerformanceLib only - describe the changes in detail in commit message
- Commit 2: Changes to Universal/Variable - describe those changes in detail in commit message

In PR description, describe both changes.